src/rpc/blockchain.cpp

```diff
@@ -2855,6 +2855,9 @@ return RPCHelpMan{
         return data;
     };
 
+    // We present the IBD/background chainstate as "normal" to users, since
+    // it should not matter to them if there is also a snapshot chainstate.
+    // To users, the IBD chainstate should always be considered "normal".
```

Comment:
I think we have a simple misunderstanding of my wording, probably my English makes the comment harder to parse than it needs to be :)

My goal is to explain to developers why we talk about a "normal" chainstate when speaking to the user (via RPC interface) while internally referring to it as ibd/background in the code. I am trying to convey the same thing you said in your comment here: https://github.com/bitcoin/bitcoin/pull/27596#discussion_r1342035181

Particularly when you say `The idea is for the "normal" chainstate to refer to a typical fully validated chainstate [...] I think the term "IBD chainstate" is basically meaningless because both chainstates are downloaded during IBD.` I am trying to say the same thing here: The normal chainstate is the ibd chainstate, no matter if there is a snapshot chainstate or not. Hence "it does not matter" that there is a snapshot with regards to the naming of the ibd/background/normal chainstate from the user perspective.

I am happy to take suggestions for better wording but I don't want to just explain what the code does. I want to document the internal/external divergence in the naming of the chainstates for future developers.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/28562#discussion_r1344824078

I implemented the changes I suggested above in #28590, so will mark this thread resolved to avoid unnecessarily slowing down this PR.